packages/build/tests/plugins/validate/snapshots/tests.js.md

```diff
@@ -110,18 +116,21 @@ Generated by [AVA](https://ava.li).
     > Loading plugins␊
     ␊
     ┌─────────────────────────────┐␊
-    │     Netlify Build Error     │␊
+    │        Build failed         │␊
     └─────────────────────────────┘␊
     ␊
-    Error loading "/file/path" plugin:␊
-    Error: Invalid property 'invalid'.␊
-    Please use a property name. One of:␊
-     - name␊
-     - scopes␊
-     - config␊
+    > Error message␊
+      Error: Invalid property 'invalid'.␊
+      Please use a property name. One of:␊
+       - name␊
+       - scopes␊
```

Comment:
Likewise to removing `api` from plugin functions, we need to remove the `scopes` field as well until we re-introduce the `api` feature